etc/parse_libs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+## create json for all files
+
+$1 -f $2 | awk -F " " '{print "[\"" $1 "\",\"" $3 "\"]\,"}' | tail -n+4
```

Comment:
Please add trailing new line. 

Also, I don't see how this is simpler than doing it in Julia directly? All `awk` does here is to split each line at spaces, which is trivial to do in julia; and then it assembles it together into some JSON.

I don't know right now what the input data to be parsed here looks like, but  if you give me a sample input and output, I'd be happy to provide a Julia version of it.